asterius/rts/rts.mjs

```diff
@@ -220,6 +220,7 @@ export async function newAsteriusInstance(req) {
 
     Object.assign(__asterius_exports, __asterius_wasm_instance.exports);
     __asterius_exports.hs_init();
+    __asterius_exports.__wasm_apply_relocs();
```

Comment:
We call `__wasm_apply_relocs` as an exported function, but it seems it hasn't been declared as a function export anywhere?

Reply:
It is declared as part of `rtsFunctionExports` here: https://github.com/tweag/asterius/pull/750/files#diff-52a18eb8b68ef165a247a8fc8ebf536bR683